lib/matplotlib/tests/test_streamplot.py

```diff
@@ -34,6 +34,18 @@ def test_startpoints():
     plt.plot(start_x, start_y, 'ok')
 
 
+@image_comparison(['streamplot_startpoints_no_broken'], remove_text=True,
```

Comment:
We try to limit our image tests as they take substantial repo size.  I would just choose your favourite of these, versus having all of them test this new functionality.

Reply:
I picked out the one with maxlength, as we may want that to raise an exception if we think this should conflict with it.